src/test/java/com/ibm/cloud/sdk/core/test/model/generated/Animal.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2019.
```

Comment:
I copied in new versions of the Animal-related classes from the generator test snapshots in order to get the Builder classes that were previously introduced for dynamic models.